pkg/pipelines/triggers/pipelinerun_test.go

```diff
@@ -21,7 +21,7 @@ func TestCreateDevCDPipelineRun(t *testing.T) {
 			Resources:          createDevResource("REPLACE_IMAGE:$(params.gitref)"),
 		},
 	}
-	template := createDevCDPipelineRun(sName)
+	template := createDevCDPipelineRun(sName, "REPLACE_IMAGE")
```

Comment:
This is probably not an indicative value for the parameter?

how about...

```suggestion
	template := createDevCDPipelineRun(sName, "example.com:5000/testing/testing")
```

Reply:
Fixed.